persistence/src/main/java/se/inera/intyg/intygsadmin/persistence/entity/BannerEntity.java

```diff
@@ -44,7 +45,7 @@ public class BannerEntity {
 
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
-    @Type(type = "uuid-char")
+    @JdbcTypeCode(SqlTypes.CHAR)
```

Comment:
Would like to understand the usage of this as I haven't seen it before. Not that I doubt it, just of curiosity.